CITATION.cff

```diff
@@ -14,6 +14,7 @@ authors:
     given-names: "Yu"
     email: "Tony.Xiang@Alliander.com"
     affiliation: "Alliander"
+    orcid: "https://orcid.org/0000-0001-5889-0067"
```

Comment:
What is this and why is it necessary?

Reply:
orcid is a unique researcher ID. you can register yourself and you can link to e.g. your existing papers. while not strictly necessary, since CFF now supports it, we might as well add it